agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,6 +123,58 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
```

Comment:
is this getconf.py part of pbench utils ?!
also looks like we call it many times, we can do something like single call with arguments and return some sort of json string with with answer per argument ?